### PR TITLE
update position and velocity controller to conventions + additional information

### PR DESCRIPTION
**Pull request by heappy**
Multiple changes to controller diagrams - changed to conventions

1. Position controller + additional information (e.g. parameter for gains)
2. Velocity controller + additional information (e.g. parameter for gains) and removed saturation
3. Added new section for Acceleration conversion to thrust and attitude setpoint - refering to `ControlMath` and added explanation from @bresch 

Changed file format from from .jpg to .png 
New files added here:
- mc_position_diagram.png
- mc_velocity_diagram.png
Removed files:
- mc_position_diagram.jpg
- mc_velocity_diagram.jpg

Please review changes @bresch and @hamishwillee

### CODE REVIEW

File: en/flight_stack/controller_diagrams.md [hunk 1]
```diff
@@ -46,20 +46,34 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
...
 * The rate command is saturated.
 
+### Multicopter Acceleration to Thrust and Attitude setpoint conversion
+* The acceleration setpoints generated by the velocity controller will be converted to thrust and attitude setpoints
+* Implementation details can be found in `ControlMath.cpp`
+* Converted acceleration setpoints will be saturated and priotized in vertical and horizontal thrust. 
+* Thrust saturation is done after computing the corresponding thrust:
+ 1. compute required vertical thrust (`thrust_z`)
+ 2. saturate `thrust_z` with `MPC_THR_MAX`
+ 3. saturate `thrust_xy` with `(MPC_THR_MAX^2 - thrust_z^2)^0.5`
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md:56]:
```suggestion
### Multicopter Acceleration to Thrust and Attitude Setpoint Conversion

* The acceleration setpoints generated by the velocity controller will be converted to thrust and attitude setpoints.
* Converted acceleration setpoints will be saturated and prioritized in vertical and horizontal thrust. 
* Thrust saturation is done after computing the corresponding thrust:
   1. Compute required vertical thrust (`thrust_z`)
   1. Saturate `thrust_z` with `MPC_THR_MAX`
   1. Saturate `thrust_xy` with `(MPC_THR_MAX^2 - thrust_z^2)^0.5`
```

Implementation details can be found in `ControlMath.cpp`.

File: en/flight_stack/controller_diagrams.md [hunk 2]
```diff
@@ -46,20 +46,34 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
...
+
 ### Multicopter Velocity Controller
 
-![MC Velocity Control Diagram](../../assets/diagrams/mc_velocity_diagram.jpg)
+![MC Velocity Control Diagram](../../assets/diagrams/mc_velocity_diagram.png)
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md:60]: FWIW Personally I prefer consistency in diagram file type, and generally PNG results in cleaner images for similar file sizes. So I like this. Note though, there is no rule on this.

**Comment by hamishwillee** [comment]
@bresch This looks OK to be "editorially" but it needs your technical review. Thanks @heappy

File: en/flight_stack/controller_diagrams.md [hunk 3]
```diff
@@ -46,20 +46,36 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
...
+### Multicopter Acceleration to Thrust and Attitude Setpoint Conversion
+
+* The acceleration setpoints generated by the velocity controller will be converted to thrust and attitude setpoints.
+* Converted acceleration setpoints will be saturated and prioritized in vertical and horizontal thrust. 
+* Thrust saturation is done after computing the corresponding thrust:
+   1. Compute required vertical thrust (`thrust_z`)
+   1. Saturate `thrust_z` with `MPC_THR_MAX`
+   1. Saturate `thrust_xy` with `(MPC_THR_MAX^2 - thrust_z^2)^0.5`
+   
+Implementation details can be found in `ControlMath.cpp`.
```
Comment (bresch) [inline en/flight_stack/controller_diagrams.md:58]:
```suggestion
Implementation details can be found in `PositionControl.cpp` and `ControlMath.cpp`.   
```

File: en/flight_stack/controller_diagrams.md [hunk 4]
```diff
@@ -46,20 +46,36 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
...
 
 ### Multicopter Position Controller
 
-![MC Position Control Diagram](../../assets/diagrams/mc_position_diagram.jpg)
+![MC Position Control Diagram](../../assets/diagrams/mc_position_diagram.png)
 
-* Simple P controller that commands a velocity.
-* The commanded velocity is saturated to keep the velocity in certain limits.
+* Simple P controller that commands a velocity. 
+* The commanded velocity is saturated to keep the velocity in certain limits. See parameter `MPC_XY_VEL_MAX`. This parameter sets the maximum possible horizontal velocity. This differs from `MPC_XY_CRUISE` in autonomous flight modes.
```
Comment (bresch) [inline en/flight_stack/controller_diagrams.md:75]:
```suggestion
* The commanded velocity is saturated to keep the velocity in certain limits. See parameter `MPC_XY_VEL_MAX`. This parameter sets the maximum possible horizontal velocity. This differs from the maximum **desired** speed `MPC_XY_CRUISE` (autonomous modes) and `MPC_VEL_MANUAL` (manual position control mode).
```

**Comment by hamishwillee** [comment]
Thanks very much both of you.

**Pull request merged**